testsuite/run_forge.sh

```diff
@@ -168,15 +169,19 @@ if [ -n "$AVG_TPS" ]; then
     echo "forge_job_avg_tps {FORGE_CLUSTER_NAME=\"$FORGE_CLUSTER_NAME\",FORGE_NAMESPACE=\"$FORGE_NAMESPACE\",GITHUB_RUN_ID=\"$GITHUB_RUN_ID\"} $AVG_TPS" | curl -u "$PUSH_GATEWAY_USER:$PUSH_GATEWAY_PASSWORD" --data-binary @- ${PUSH_GATEWAY}/metrics/job/forge
     if [[ "$AVG_TPS" -lt "$TPS_THRESHOLD" ]]; then
         echo "(\!) AVG_TPS: ${avg_tps} < ${TPS_THRESHOLD} tps"
-        FORGE_EXIT_CODE=1
+	if [ "$FORGE_RUNNER_MODE" != "local" ]; then
```

Comment:
Can we move this perf threshold checking into the performance test itself ?

Reply:
We can do that, but that will require some plumbing in the transaction emitter. May be as a follow up I can take a look at this?

Reply:
hm ok yea. this script is just getting too complicated